app/connectors/ArrivalConnector.scala

```diff
@@ -32,9 +33,12 @@ import scala.concurrent.Future
 
 class ArrivalConnector @Inject()(http: HttpClient, appConfig: AppConfig) extends BaseConnector {
 
-  def get(arrivalId: ArrivalId)(implicit requestHeader: RequestHeader, hc: HeaderCarrier, ec: ExecutionContext): Future[HttpResponse] = {
+  def get(arrivalId: ArrivalId)(implicit requestHeader: MessageRequest[_], hc: HeaderCarrier, ec: ExecutionContext): Future[HttpResponse] = {
```

Comment:
if you are going to change the type, can you change the name please

Reply:
To be honest, it is probably better that the channelType (as it is the only reason for this change) is passed in as an explicit parameter.